Documentation/Manual/Texinfo/docstrings/fun-clouseau-format-place-cells.texi

```diff
@@ -0,0 +1,29 @@
+@anchor{Function clouseau:format-place-cells}
+@deffn {Function} {@symbol{format-place-cells,clouseau}} stream object place-class cell &key label label-style place-style object-style
+Present @var{object}'s child selected by @var{place-class} and @var{cell} to @var{stream}.
+
+Retrieve an existing place instance for @var{object}'s child selected by
+@var{place-class} and @var{cell} or make a new instance of
+@var{place-class}.
+
+Each of the following is written to @var{stream} within a separate
+@func{clim:formatting-cell}:
+
+@enumerate
+@item
+If non-@t{nil}, @var{label} is written to @var{stream} with
+@var{LABEL-STYLE}, if non-NIL, in effect.  If @var{label} is a function,
+it is called with @var{stream} as its sole argument.  The label is
+presented with @var{label-style}, if non-@t{nil}, in effect.
```

Comment:
"if non-nil, in effect" is a bit unusual in several places around here.

Reply:
Change.